scalarm_client_csharp/Experiment.cs

```diff
@@ -357,11 +357,17 @@ public static IList<ValuesMap> ConvertTypes(IList<ValuesMap> results)
 			foreach (var record in convertedResults) {
                 // http://stackoverflow.com/a/18288740/1367361
                 List<string> keys = new List<string>(record.Keys);
-                foreach (string key in keys)
-                {
-                    // TODO: check with string values - probably there will bo problem with deserializing because lack of ""
-                    record[key] = JsonConvert.DeserializeObject(record[key].ToString());
-                }
+				foreach (string key in keys) {
+
+					// TODO: check with string values - probably there will bo problem with deserializing because lack of ""
+					try {
+						record [key] = JsonConvert.DeserializeObject(record[key].ToString());
+					} catch (Newtonsoft.Json.JsonReaderException e) {
+					// exception when evaluate toString on String object -> wrong Deserialize
```

Comment:
@Andriu01 - napisać komentarz, że w takim przypadku zostawiamy to jako string, bo to najpewniej jest zwyczajny string